asyncpraw/models/user.py

```diff
@@ -107,18 +105,20 @@ def karma(self) -> Dict[Subreddit, Dict[str, int]]:
             order to retrieve the values, preferably through ``dict.items()``.
         """
         karma_map = {}
-        for row in self._reddit.get(API_PATH["karma"])["data"]:
+        response = await self._reddit.get(API_PATH["karma"])
+        for row in response["data"]:
```

Comment:
Did you test this? I thought response was already the `data`

Reply:
Yes I have tested this. The response returns the following JSON:

```json
{
    "kind": "KarmaList",
    "data": [
        {"sr": "pics", "comment_karma": 24, "link_karma": 7172}
    ]
}
```